diesel.toml

```diff
@@ -0,0 +1,5 @@
+# For documentation on how to configure this file,
```

Comment:
looks like migration and diesel file added to the root from some init command of desiel. 

We have `./diesel.sh` to wrap the cli and help working with the workspaces.